orchagent/bufferorch.cpp

```diff
@@ -58,9 +58,20 @@ void BufferOrch::initBufferReadyList(Table& table)
 {
     SWSS_LOG_ENTER();
 
+    // init all ports with an empty list
+    for (const auto& it: gPortsOrch->getAllPorts())
```

Comment:
I think gPortsOrch->getAllPorts() will return only one port, the CPU port. Is it what you expected?